maven-surefire-common/src/main/java/org/apache/maven/surefire/report/RunStatistics.java

```diff
@@ -100,13 +102,55 @@ public synchronized RunResult getRunResult()
 
     public synchronized String getSummary()
     {
-        String summary =
-            "Tests run: " + completedCount + ", Failures: " + failures + ", Errors: " + errors + ", Skipped: "
-                + skipped;
+        return getSummary( completedCount, failures, errors, skipped, flakes );
+    }
+
+    public static String getSummary( int completedCount, int failures, int errors, int skipped, int flakes )
+    {
+        MessageBuilder builder = MessageUtils.buffer();
+
+        builder.a( "Tests " );
+
+        if ( completedCount == 0 )
+        {
+            builder.a( "run: 0" );
+        }
+        else
+        {
+            builder.success( "run: " + completedCount );
```

Comment:
Thank you for the inspiration. I took your code but changes the colors in "Tests run: ". In my case "Tests run: <num>" is green if the test class is without errors/failures/skipped. And "Tests" red or yellow depending on the status. The next parts like Failures of Errors are displayed as in you case the same. Bold class name as in your case.
I hope you will enjoy it with me.
One remark which is very important is the fact that Maven 2.2.1 and 3.0.x should not be used as minimum prerequisite because the Mojo Logger has sme error because `Log.info(string)` is not displayed like this:
`[INFO] string`
but in new line:
`[INFO]`
` string`
If you know this issue and treatment, pls help me. It was not visible how to fix it using Mojo Logger and therefore I wrote prerequisite Maven 3.1.x or higher. These prerequisites are important during the Vote because the ITs in Surefire project fail with Maven 3.0.x or prior to.